test/unit/tests/utils/transaction.test.ts

```diff
@@ -162,13 +163,22 @@ describe('transaction utils', () => {
           });
         });
 
+      // Mock txs/:id/cbor responses
+      nock(/.+blockfrost.io/)
+        .persist()
+        .get(/api\/v0\/txs\/.*\/cbor/)
+        .delay(1000)
+        .reply(200, (_uri, _body, cb) => {
+          cb(null, { cbor: '0123456789abcdef0123456789abcdef0123456789abcdef' });
+        });
+
       let promises = [];
 
       // run thousands of fetchTransactionData
       // (promise limiter concurrency set to 500 by default)
       for (let i = 0; i < 5000; i++) {
         promises.push(
-          transactionUtils.fetchTransactionData(`txHash-${i}`).then(data => {
+          transactionUtils.fetchTransactionData(`txHash-${i}`, Math.random() > 0.3).then(data => {
```

Comment:
```suggestion
          transactionUtils.fetchTransactionData(`txHash-${i}`, true).then(data => {
```

Let's just enable it for all calls. Introducing randomness won't make the test more useful and could just make it flaky. We could add another test if it make sense to test `cbor=false` param, but I think we are good.

Reply:
7d3b2b9 should solve this